### PR TITLE
Remove Java 5 references and clean up

### PR DESCRIPTION
**Pull request by elharo**
Mostly a copy edit for clarity. @slawekjaranowski

### CODE REVIEW

File: src/site/apt/toolchains/custom.apt [hunk 1]
```diff
@@ -25,22 +25,22 @@
 
 Custom Toolchains
 
-  You can create your own custom toolchains with plugins using them.
+  You can create your own custom toolchains for plugins to use.
```
Comment (hboutemy) [inline src/site/apt/toolchains/custom.apt:28]: in fact, you need in 2 steps:
1. create a toolchain
2. have a plugin use it

Comment (slawekjaranowski) [reply src/site/apt/toolchains/custom.apt:28]: agree with @hboutemy - old sentence was more accurate for me also

Comment (gnodet) [reply src/site/apt/toolchains/custom.apt:28]: @elharo I'm thinking about release a new version of this plugin. Do you want to remove that line and merge the PR ?

Comment (gnodet) [reply src/site/apt/toolchains/custom.apt:28]: maybe `You can create your own custom toolchains to use with standard plugins (such as maven-compiler-plugin) or custom plugins` ?

Comment (elharo) [reply src/site/apt/toolchains/custom.apt]: done, though I'm not sure "custom plugin" is really a thing. At this level, aren't they all just plugins?

**Review by slachiewicz — APPROVED** [review]

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**